macro/README.md

```diff
@@ -1,66 +1,23 @@
-# Iroha Macro
+# Iroha Macros
 
-## Description
+This crate contains macros and attributes for Iroha projects:
 
-This crate contains macros and attributes for Iroha projects.
-`iroha_derive` contains derive macros.
-
-### Features
-
-* `log` attribute for debugging functions inputs and output
+- `iroha_derive` is a custom derive macros for Iroha
+- `log` attribute is used for debugging function inputs and output
```

Comment:
I don't really see this `log` attribute in code so I'm not sure about this line

Reply:
Remove it. We found that the `log` attribute causes deadlocks if not used carefully.

Reply:
removed